index.html

```diff
@@ -20,10 +20,16 @@
         <li class="cellular"><img src="assets/Bars/Status Bars/Cellular Connectionmobile-top.png"></li>
         <li class="wifi"><img src="assets/Wifiwifi.png"></li>
         <li class="battery"><img src="assets/Bars/Status Bars/Batterymobile-top.png"></li>
+      </div><br><br>
+      <div class="nav">
+        <h1 class="my-logo">My logo</h1>
+        <img src="assets/Unionnavbar.png" class="navbar">
```

Comment:
- [ ] I would kindly suggest you wrap your `hamburger` icon inside a `nav` tag, The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices. 👍 👍